Package.swift

```diff
@@ -37,9 +37,10 @@ let package = Package(
         .target(
             name: "RSDKUtilsNimble", 
             dependencies: ["RSDKUtilsMain", "Nimble"]),
+        .target(name: "REventLogger"),
```

Comment:
`REventLogger` should also be added to products array as library in `Package.swift`.